src/Raven.Server/Documents/Handlers/Processors/Studio/AbstractStudioCollectionHandlerProcessorForDeleteCollection.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Raven.Server.Json;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Handlers.Processors.Studio
+{
+    internal abstract class AbstractStudioCollectionHandlerProcessorForDeleteCollection<TRequestHandler, TOperationContext> : AbstractDatabaseHandlerProcessor<TRequestHandler, TOperationContext>
+        where TOperationContext : JsonOperationContext
+        where TRequestHandler : AbstractDatabaseRequestHandler<TOperationContext>
+    {
+        protected AbstractStudioCollectionHandlerProcessorForDeleteCollection([NotNull] TRequestHandler requestHandler) : base(requestHandler)
+        {
+        }
+
+        protected abstract ValueTask DeleteCollectionAsync(TOperationContext context, IDisposable returnContextToPool, string collectionName, HashSet<string> excludeIds, long operationId);
+
+        protected abstract long GetNextOperationId();
+
+        public override async ValueTask ExecuteAsync()
+        {
+            var returnContextToPool = ContextPool.AllocateOperationContext(out TOperationContext context);
+
+            var collectionName = RequestHandler.GetStringQueryString("name");
+            var operationId = RequestHandler.GetLongQueryString("operationId", required: false) ?? -GetNextOperationId();
```

Comment:
remove `-` 
It is no longer required, we generate correct numbers now.

Reply:
Done. But I think we might want to change this to create an operation on the orchestrator as well? as done in https://github.com/ravendb/ravendb/pull/14314